datasetUtils/checkForServiceAvailability.go

```diff
@@ -28,48 +26,66 @@ type ServiceAvailability struct {
 	Qa         OverallAvailability
 }
 
+var GitHubMainLocation = "https://raw.githubusercontent.com/paulscherrerinstitute/scicat-cli/main"
+
+// CheckForServiceAvailability checks the availability of the dataset ingestor service.
+// It fetches a YAML file from GitHubMainLocation, parses it, and logs the service availability status.
 func CheckForServiceAvailability(client *http.Client, testenvFlag bool, autoarchiveFlag bool) {
-	resp, err := client.Get(DeployLocation + "datasetIngestorServiceAvailability.yml")
+	s, err := getServiceAvailability(client)
 	if err != nil {
-		fmt.Println("No Information about Service Availability")
+		log.Printf("Failed to get service availability: %v", err)
 		return
```

Comment:
This passes all errors on to the user. Are these generally suitable error messages, or will this result in confusing errors. This is not critical code, so I don't think we need to print long scary errors (#22).

Reply:
It is better to have these error messages than not so that one can see where the problem was.
`log.Printf` prints the error message to `stderr`, whereas `fmt.Println` prints to `stdout`.